src/app/catalog/product/components/product-master/product-master.component.ts

```diff
@@ -59,25 +60,4 @@ export class ProductMasterComponent implements OnInit, AfterViewInit {
   public isContentInTheViewport(index: number): boolean {
     return this.displayableContentSections[index / 3];
   }
-
-  public ngAfterViewInit(): void {
```

Comment:
Just curious: Is this workaround not needed anymore?
Is this fixed automatically by the new angular version?
Or is it the `takeUntilDestroyed` that's doing the trick here?